internal/compiler/generator/cpp.rs

```diff
@@ -579,7 +579,8 @@ pub fn generate(doc: &Document) -> impl std::fmt::Display {
     }
 
     for ty in doc.root_component.used_types.borrow().structs.iter() {
-        if let Type::Struct { fields, name: Some(name), node: Some(node) } = ty {
+        // FIXME: add feature as deriven
```

Comment:
I think it's fine to just ignore this in C++ since there is no such things there.  So no need for this comment